llvm/test/CodeGen/AMDGPU/inline-attr.ll

```diff
@@ -1,6 +1,6 @@
-; RUN: opt -mtriple=amdgcn--amdhsa -S -O3 -enable-unsafe-fp-math %s  | FileCheck -check-prefix=GCN -check-prefix=UNSAFE %s
-; RUN: opt -mtriple=amdgcn--amdhsa -S -O3 -enable-no-nans-fp-math %s | FileCheck -check-prefix=GCN -check-prefix=NONANS %s
-; RUN: opt -mtriple=amdgcn--amdhsa -S -O3 -enable-no-infs-fp-math %s | FileCheck -check-prefix=GCN -check-prefix=NOINFS %s
```

Comment:
This change depends on whether we want to keep the check lines previously generated by AMDGPUAttributor. It doesn't look like an AMDGPUAttributor related test case.

Reply:
I'm surprised the attributor induced attributes are here in the first place. I would just leave this running non-LTO (also this test doesn't really belong in codegen. I'm also not sure why its running -O3 instead of just the inliner)